pkg/kubelet/apis/config/types.go

```diff
@@ -75,6 +75,20 @@ const (
 	PodTopologyManagerScope = "pod"
 )
 
+type GracefulNodeShutdownPodPolicyType string
+
+const (
+	// GracefulNodeShutdownPodPolicySetTerminal is Graceful Node Shutdown policy type,
+	// which will ensure that during graceful node shutdown,
+	// pods will be placed into terminal phase and will start again if nodes becomes ready again.
+	GracefulNodeShutdownPodPolicySetTerminal = "SetTerminal"
+
+	// GracefulNodeShutdownPodPolicyLeaveRunning is Graceful Node Shutdown policy type,
+	// which will ensure that during graceful node shutdown,
+	// pods will be left running and will start again if the if nodes becomes ready again.
```

Comment:
should the docs distinguish between what is happening to the process and what is marked in the API?

```suggestion
	// pods will be left marked as running and will start again if the node becomes ready again.
```

Reply:
Good suggestion, I rephrased to emphaize that kubelet will terminate the pods in both cases, and the status to API server is different depending between `SetTerminal` `LeaveRunning`